src/scripts/main.js

```diff
@@ -1,3 +1,65 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+appendRow.addEventListener('click', () => {
+  const rows = table.rows;
+
+  removeRow.disabled = false;
+
+  if (rows.length < 10) {
+    const row = rows[0];
```

Comment:
Ensure that the table has at least one row before accessing `rows[0]`. If the table is empty, this will throw an error.